Soccar [UI]/src/nl/soccar/ui/fx/controller/GameControlsFXMLController.java

```diff
@@ -0,0 +1,35 @@
+package nl.soccar.ui.fx.controller;
+
+import java.net.URL;
+import java.util.ResourceBundle;
+import java.util.logging.Logger;
+import javafx.fxml.FXML;
+import javafx.fxml.Initializable;
+import javafx.scene.control.Button;
+import nl.soccar.library.Player;
+import nl.soccar.ui.Main;
+import nl.soccar.ui.fx.FXMLConstants;
+
+/**
+ * FXML Controller class
+ *
+ * @author PTS34A
+ */
+public class GameControlsFXMLController implements Initializable {
+
+    @FXML
+    private Button btnLogOut;
+    @FXML
+    private Button btnReturn;
+
+    private static final Logger LOGGER = Logger.getLogger(GameControlsFXMLController.class.getSimpleName());
+
+    private Player currentPlayer;

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "currentPlayer" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AS1068)